components/indexer/component.go

```diff
@@ -188,6 +189,8 @@ func run() error {
 
 		Component.LogInfo("Starting API server ...")
 
+		// setup the `/health` route
+		setupRoutes()
```

Comment:
I wonder if we maybe should add it to the "normal" API itself? So inside the IndexerServer.